util/dvsim/Deploy.py

```diff
@@ -216,7 +243,11 @@ def is_equivalent_job(self, item):
         return True
 
     def dispatch_cmd(self):
-        self.exports.update(os.environ)
+        # Update the shell's env vars with self.exports. Values is exports must
```

Comment:
Nit: I think you probably mean "Values **in** exports"?

Reply:
Fixed.